python/lib/core/pyproject.toml

```
@@ -0,0 +1,26 @@
+[build-system]
+requires = ["setuptools >= 70.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "dmod.core"
+authors = [
+    { name = "Robert Bartel" },
+    { name = "Austin Raney", email = "austin.raney@noaa.gov" },
+    { name = "Chris Tubbs" },
+]
+dependencies = ["pydantic>=1.10.8,~=1.10", "typing_extensions"]
+readme = "README.md"
+description = ""
```

Might as well add something here too.

```suggestion
description = "Library package with core DMOD concrete and abstract types depended upon by other DMOD libraries."
```